http-core/src/main/scala/org/apache/pekko/http/impl/engine/http2/Http2Blueprint.scala

```diff
@@ -198,6 +199,20 @@ private[http] object Http2Blueprint {
       Flow[FrameEvent].map(FrameRenderer.render).prepend(Source.single(Http2Protocol.ClientConnectionPreface)),
       Flow[ByteString].via(new Http2FrameParsing(shouldReadPreface = false, log)))
 
+  private def rapidResetMitigation(
+      settings: Http2ServerSettings): BidiFlow[FrameEvent, FrameEvent, FrameEvent, FrameEvent, NotUsed] = {
+    def frameCost(event: FrameEvent): Int = event match {
+      case _: FrameEvent.DataFrame         => 0
+      case _: FrameEvent.WindowUpdateFrame => 0 // TODO: should we throttle these?
+      case _                               => 1
+    }
```

Comment:
this can be contantFunc

Reply:
I don't know what `constantFunc` means. Do you mean making this a `val` instead of a `def`?

Reply:
Yes, make just like the `Keep.left/right`